crates/cxx-qt-lib/src/core/qanystringview.rs

```diff
@@ -60,6 +61,10 @@ mod ffi {
         #[doc(hidden)]
         #[rust_name = "QAnyStringView_len"]
         fn qanystringviewLen(string: &QAnyStringView) -> isize;
+
+        #[doc(hidden)]
+        #[rust_name = "QAnyStringView_to_qstring"]
+        fn toQString(string: &QAnyStringView) -> QString;
```

Comment:
There is already a `to_string` function on QAnyStringView (see line 34) which you can use.